src/EFTools/EntityDesign/VisualStudio/ModelWizard/engine/ModelGenerator.cs

```diff
@@ -87,15 +87,16 @@ internal virtual EntityStoreSchemaGeneratorDatabaseSchemaLoader CreateDbSchemaLo
             return
                 new EntityStoreSchemaGeneratorDatabaseSchemaLoader(
                     connection,
-                    storeSchemaModelVersion);
+                    storeSchemaModelVersion,
+                    _settings.FnGetLocalAppSetting());
         }
 
         // internal virtual for testing
         internal virtual SimpleMappingContext CreateMappingContext(EdmModel storeModel)
         {
             Debug.Assert(storeModel != null, "storeModel != null");
 
-            return
+            return 
```

Comment:
nit: unnecessary spacing in new diff here and in [EntityStoreSchemaGeneratorDatabaseSchemaLoader.cs](https://github.com/dotnet/ef6tools/pull/75/files#diff-6a02f43b6fdfdc61f145e06c1c68929a1ae51b9c0718fa5fb593ad0c0f8f417c) and on the lines containing `[Fact]` in [EntityStoreSchemaGeneratorDatabaseSchemaLoaderTests.cs](https://github.com/dotnet/ef6tools/pull/75/files#diff-ab79adeb822b9bcdcabaccfbb311c43fad967b4fc60628a360c34f89d2230167)

Reply:
"Trim Whitespace" was set in the diff viewer in VS.  Fixed with format document.